exchanges/persisted/src/persistedExchange.ts

```diff
@@ -159,11 +165,9 @@ export const persistedExchange =
                 sha256Hash,
               },
             };
-            if (
-              persistedOperation.kind === 'query' &&
-              preferGetForPersistedQueries
```

Comment:
Removing the check on `preferGetForPersistedQueries` allows `persistedOperation.context.preferGetMethod` to be set to `false`.